content/warnings/dont-call-proptypes.md

```diff
@@ -4,47 +4,48 @@ layout: single
 permalink: warnings/dont-call-proptypes.html
 ---
 
-> Note:
+> Uwaga:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> Z wersją React v15.5 `React.PropTypes` zostało przeniesione do innej paczki. Zamiast importować z paczki Reacta, proszę używaj [biblioteki `prop-types`](https://www.npmjs.com/package/prop-types).
 >
->We provide [a codemod script](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) to automate the conversion.
+> Dla ułatwienia migracji przygotowaliśmy [skrypt codemod](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes).
 
-In a future major release of React, the code that implements PropType validation functions will be stripped in production. Once this happens, any code that calls these functions manually (that isn't stripped in production) will throw an error.
+W przyszłych wersjach Reacta kod, który implementuje funkcje walidujące PropTypes będzie wycinany na produkcji. Gdy to nastąpi, każdy kod wywoujący te funkcje bezpośrednio będzie powodował błąd (o ile także nie zostanie wycięty na produkcji).
 
-### Declaring PropTypes is still fine {#declaring-proptypes-is-still-fine}
+### Definiowanie PropTypes pozostaje prawidłowe {#declaring-proptypes-is-still-fine}
```

Comment:
```suggestion
### Definiowanie PropTypes pozostaje prawidłowe {#declaring-proptypes-is-still-fine}
```
W języku angielskim używa się specyficznego szyku zdania, który nie zawsze dobrze brzmi po polsku. Poza tym: o co chodzi z "pozostaje prawidłowe"? Prędzej "Nadal można definiować PropTypes".

Reply:
to żeby być super klarownym może "Nadal można deklarować PropTypes w tradycyjny sposób"?